.travis.yml

```diff
@@ -80,7 +99,14 @@ before_install:
   - mkdir -p $HOME/Downloads $HOME/Deps
 
 install:
-  - travis_wait 30 ./testing/dependencies/install_dependencies.sh
+  - ./testing/dependencies/travis/install.sh
+  - ./testing/dependencies/travis/cmake.sh
+  - ./testing/dependencies/travis/ninja.sh
+  - if [[ "$ANACONDA_TESTS_ONLY" == 1 ]]; then
+      ./testing/dependencies/travis/anaconda.sh;
```

Comment:
If Anaconda install is cached, then it's not a huge problem to always run the installation, rather than have this test.

Reply:
Good point I will remove the if-check.